### PR TITLE
[FIX] account_credit_control: missing permission

### PR DESCRIPTION
**Pull request by cubells**
User with Accounting & Finance permissions got an error if he doesn't belongs to a Credit Control group on viewing partner form.

cc @Tecnativa

### CODE REVIEW

File: account_credit_control/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Copyright 2017 Okia SPRL (https://okia.be)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {'name': 'Account Credit Control',
- 'version': '10.0.1.1.0',
+ 'version': '10.0.1.1.1',
```
Comment (pedrobaeza) [inline account_credit_control/__manifest__.py]: Better to change middle number (10.0.1.2.0)

**Review by pedrobaeza — APPROVED** [review]
Once changed the version number

**Comment by pedrobaeza** [comment]
@chienandalu @luismontalba please review

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Review by chienandalu — APPROVED** [review]

**Pull request merged**